steps/AddLang.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+using NUnit.Framework;
+
+namespace MarsQA_1.Feature
+{
+    [Binding]
+    class AddLang

```

Comment:
This class name is not meaningful enough. AddLang doesn't tell much compared to AddLanguage

Furthermore, why are you creating separate classes for every single step (AddLang, DeleteEdu, login)? So if there is 20 other steps, you are going to create 20 more classes?

Question: What are you testing? Adding Language in the Profile page, right? So I would like to suggest you to create a class specifically for the Profile page, hence you don't need to create so many classes.